spring-cloud-gcp-examples/spring-cloud-gcp-storage-resource-example/pom.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
```

Comment:
Maybe unrelated to this PR, but I'm actually wondering whether our examples should have this parent.

It would probably be better not to, just so we could show some BOM usage, too.

Reply:
Valid point, but let's handle it separately across all examples.